src/modules/Auth/components/SignInForm.tsx

```diff
@@ -49,74 +53,69 @@ export const SignInForm = () => {
         py={6}
         width="450px"
       >
-        <Box display="flex" gap={2} alignItems="center" justifyContent="center">
-          <Typography variant="h6" align="center">
-            Login
-          </Typography>
-          {isSubmitting && <CircularProgress size={20} />}
-        </Box>
-        <FormProvider<SignInUserData>
-          trigger={trigger}
-          reset={reset}
-          handleSubmit={handleSubmit}
-          control={control}
-          {...rest}
+        <Typography variant="h6" align="center">
+          Login
+        </Typography>
+        <Box>{isAuthLoading && <LinearProgress />}</Box>
+        <Typography
+          color="#ff5252"
+          sx={{
+            display: isAuthError ? 'inline' : 'none',
+            fontWeight: 500,
+          }}
         >
```

Comment:
Please, use conditional rendering instead of `display: none`
```suggestion
        {isAuthError && (<Typography
          color="#ff5252"
          sx={{
            fontWeight: 500,
          }}
        >)
        }
```